### PR TITLE
Add spanish keymaps based on default lily58

### PR DESCRIPTION
**Pull request by JuanJoEstrada**
## Description

- Add a Spanish LATAM keymaps. 
- Characters like ñ, ´, { were added.
- It was create based on default Lily58

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
Hi @JuanJoEstrada 
Would it be possible to add a readme.md file that informs the reader, with words to the affect of, for this keymap to function as intended, operating system's keyboard language must be configured/set to accommodate.

File: keyboards/lily58/keymaps/spanish_latam/config.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
```
Comment (drashna) [inline keyboards/lily58/keymaps/spanish_latam/config.h:27]:
```suggestion
```

File: keyboards/lily58/keymaps/spanish_latam/keymap.c [hunk 2]
```diff
@@ -0,0 +1,154 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/lily58/keymaps/spanish_latam/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/lily58/keymaps/spanish_latam/keymap.c [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+enum layer_number {
+  _QWERTY = 0,
```
Comment (drashna) [inline keyboards/lily58/keymaps/spanish_latam/keymap.c:4]: 0 isn't needed here as it starts from 0 by default. 
```suggestion
  _QWERTY,
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/lily58/keymaps/spanish_latam/readme.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# QMK Firmware - Spanish Keymaps
+
+![QMK Logo](https://avatars.githubusercontent.com/u/7308757?s=200&v=4)
```
Comment (drashna) [inline keyboards/lily58/keymaps/spanish_latam/readme.md:3]: ?

File: keyboards/lily58/keymaps/spanish_latam/config.h [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+#define RGBLIGHT_EFFECT_RAINBOW_MOOD
+#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+#define RGBLIGHT_EFFECT_SNAKE
+#define RGBLIGHT_EFFECT_KNIGHT
+#define RGBLIGHT_EFFECT_CHRISTMAS
+#define RGBLIGHT_EFFECT_STATIC_GRADIENT
+#define RGBLIGHT_EFFECT_RGB_TEST
+#define RGBLIGHT_EFFECT_ALTERNATING
+#define RGBLIGHT_EFFECT_TWINKLE
+#define RGBLED_NUM 27
```
Comment (drashna) [inline keyboards/lily58/keymaps/spanish_latam/config.h:40]: 27 on each side, correct?

File: keyboards/lily58/keymaps/spanish_latam/readme.md (hunk 4 above)
Comment (JuanJoEstrada) [reply keyboards/lily58/keymaps/spanish_latam/readme.md]: Deleted

File: keyboards/lily58/keymaps/spanish_latam/config.h (hunk 5 above)
Comment (JuanJoEstrada) [reply keyboards/lily58/keymaps/spanish_latam/config.h:40]: Think so, I copied it from lily58\keymaps\default\config.h

**Review by drashna — APPROVED** [review]

**Comment by zvecr** [comment]
Unfortunately since this PRs creation, QMK has [changed its stance](https://docs.qmk.fm/#/ChangeLog/20230827?id=notable-changes) on accepting user keymaps to the repository.

> qmk/qmk_firmware is no longer accepting PRs for keymaps other than for manufacturer-supported keymaps. User keymap workflow has been documented [here](https://docs.qmk.fm/#/newbs) for several years. This change is to progressively reduce the maintenance burden on the project, and to allow us to focus on the core features of QMK.

In the near future we will be providing a more user focused workflow for keymaps, but till then the suggestion is to keep changes in your own fork.

While we appreciate the time that was taken on this PR, hopefully you understand we are closing it, so that QMK can improve in the long term.

**Pull request closed without merging**